src/test/java/com/ait/lienzo/client/widget/panel/mediators/PanelMediatorsTest.java

```diff
@@ -0,0 +1,139 @@
+import com.ait.lienzo.client.core.mediator.MouseWheelZoomMediator;
+import com.ait.lienzo.client.core.shape.Layer;
+import com.ait.lienzo.client.core.shape.Scene;
+import com.ait.lienzo.client.core.shape.Viewport;
+import com.ait.lienzo.client.widget.panel.LienzoBoundsPanel;
+import com.ait.lienzo.client.widget.panel.scrollbars.ScrollablePanel;
+import com.ait.lienzo.test.LienzoMockitoTestRunner;
+import com.ait.tooling.common.api.java.util.function.Supplier;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(LienzoMockitoTestRunner.class)
+public class PanelMediatorsTest {
+
+    @Mock
+    private ScrollablePanel panel;
+
+    @Mock
+    private PanelPreviewMediator previewMediator;
+
+    private PanelMediators tested;
+    private Viewport viewport;
+
+    @Before
+    public void setUp() {
+        Layer layer = new Layer();
+        Scene scene = new Scene();
+        viewport = new Viewport(scene, 1200, 1200);
+        scene.add(layer);
+        when(panel.getLayer()).thenReturn(layer);
+        tested = new PanelMediators().init(new Supplier<LienzoBoundsPanel>() {
+            @Override
+            public LienzoBoundsPanel get() {
+                return panel;
+            }
+        }, new Supplier<PanelPreviewMediator>() {
+            @Override
+            public PanelPreviewMediator get() {
+                return previewMediator;
+            }
+        });
+    }
+
+    @Test
+    public void testInit() {
+        assertNotNull(tested.getZoomMediator());
+        MouseWheelZoomMediator zoomMediator = tested.getZoomMediator();
+        assertEquals(PanelMediators.MIN_SCALE, zoomMediator.getMinScale(), 0d);
+        assertEquals(PanelMediators.MAX_SCALE, zoomMediator.getMaxScale(), 0d);
+        assertEquals(PanelMediators.ZOOM_FACTOR, zoomMediator.getZoomFactor(), 0d);
+        assertFalse(zoomMediator.isEnabled());
+        assertNotNull(tested.getPanMediator());
+        MousePanMediator panMediator = tested.getPanMediator();
+        assertTrue(panMediator.isXConstrained());
+        assertTrue(panMediator.isYConstrained());
+        assertFalse(panMediator.isEnabled());
+        assertNotNull(tested.getPreviewMediator());
+        PanelPreviewMediator previewMediator = tested.getPreviewMediator();
+        assertFalse(previewMediator.isEnabled());
+        IMediator medidator1 = viewport.getMediators().pop();
+        IMediator medidator2 = viewport.getMediators().pop();
+        assertEquals(zoomMediator, medidator1);
+        assertEquals(medidator2, panMediator);
```

Comment:
for consistency with the line above, could we please use `assertEquals(panMediator, medidator2);` ?